kademlia/network.py

```diff
@@ -73,11 +73,11 @@ async def listen(self, port, interface='0.0.0.0'):
         # finally, schedule refreshing table
         self.refresh_table()
 
-    def refresh_table(self):
+    def refresh_table(self, interval: float = 3600):
```

Comment:
There are no other typehints in the code, so would prefer to maintain consistency.